packages/reactotron-redux/src/create-action-tracker.js

```diff
@@ -40,10 +39,24 @@ export default (reactotron, trackerOptions = {}) => {
 
         var unwrappedAction = action.type === 'PERFORM_ACTION' && action.action ? action.action : action
 
-        const exceptionPattern = exceptionPatterns.reduce((prev, pattern) => pattern(unwrappedAction.type) || prev, false);
+        const checkException = (exception, actionType) => {
+          switch (typeof exception) {
+            case 'string':
+              return actionType === exception;
```

Comment:
We use the `standard` standard.  If you run `npm run lint` you'll see it won't like semi-colons.